master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -65,6 +65,7 @@ spec:
 | maxIpsetSize                       | Maximum size for the ipsets used by Felix to implement tags. Should be set to a number that is greater than the maximum number of IP addresses that are ever expected in a tag. | int | int | `1048576` |
 | metadataAddr                       | The IP address or domain name of the server that can answer VM queries for cloud-init metadata. In OpenStack, this corresponds to the machine running nova-api (or in Ubuntu, nova-api-metadata). A value of `none` (case insensitive) means that Felix should not set up any NAT rule for the metadata path.  | IPv4, hostname, none | string | `127.0.0.1` |
 | metadataPort                       | The port of the metadata server. This, combined with global.MetadataAddr (if not 'None'), is used to set up a NAT rule, from 169.254.169.254:80 to MetadataAddr:MetadataPort. In most cases this should not need to be changed. | int | int | `8775` |
+| policySyncPathPrefix               | File system path where Felix notifies services of policy changes over Unix Domain Socket(UDS). Set to `""` to disable. | string | string | `/var/run/nodeagent` |
```

Comment:
I would suggest lowercasing and perhaps pluralizing "Domain Socket". So it would become "domain sockets". Also, you should insert a space between socket and (UDS), so it would be "domain sockets (UDS)". However, since the acronym UDS remains unused elsewhere in this page, I believe you could simply omit it.